src/components/TransactionsGroup.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text, TouchableOpacity, StyleSheet, TextInput, Keyboard, Button, ActivityIndicator, Pressable, TouchableWithoutFeedback } from 'react-native';
+import { Searchbar } from 'react-native-paper';
+
+const TransactionsGroup = () => {
+    const [selection, setSelection] = useState(1);
+
+    const [searchText, setSearchText] = useState("");
+    const [searchEntered, setSearchEntered] = useState(false);
+    const [transactions, setTransactions] = useState();
+
+    //Use effect later to get transaction data
+
+    const handleSearch = () => {
+        //todo search transactions
+        console.log(searchText);
+        setSearchEntered(true);
+    }
+
+    return (
+        //Line below allows searchbar to dismiss keyboard when clicked away
+        <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}> 
+            <View style={styles.container}>
+                <View style={styles.btnGroup}>
+                    <Pressable style={[styles.btn, selection === 1 ? styles.selectedBtn : styles.unselectedBtn]} onPress={() => setSelection(1)}>
+                        <Text style={[styles.btnText, selection === 1 ? { color: '#6B7280' } : {color: '#A9A9A9'}]}>All History</Text>
+                    </Pressable>
+                    <Pressable style={[styles.btn, selection === 2 ? styles.selectedBtn : styles.unselectedBtn]} onPress={() => setSelection(2)}>
+                        <Text style={[styles.btnText, selection === 2 ? { color: '#6B7280' } : {color: '#A9A9A9'}]}>Expenses</Text>
+                    </Pressable>
+                    <Pressable style={[styles.btn, selection === 3 ? styles.selectedBtn : styles.unselectedBtn]} onPress={() => setSelection(3)}>
+                        <Text style={[styles.btnText, selection === 3 ? { color: '#6B7280' } : {color: '#A9A9A9'}]}>Earnings</Text>
+                    </Pressable>
+                </View>
+
+                <View style={styles.searchFilterContainer}>
+                    <Searchbar
+                        placeholder="Search by..."
+                        onChangeText={query => setSearchText(query)}
+                        onSubmitEditing={handleSearch}
+                        onFocus={() => setSearchEntered(false)}
+                        value={searchText}
+                        style={[styles.searchbar, searchEntered ? {backgroundColor: '#E6ECFE'} : {backgroundColor: 'white'}]}
+                    />
+                    <TouchableOpacity style={styles.filtersButton}> 
+                        <Text style={styles.filtersButtonText}>Filters (0)</Text>
```

Comment:
Just reminder that we want to use pressables instead of touchable opacities (you know that already)
Switch this over when u get the chance